kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiTHttpFrontendService.scala

```diff
@@ -215,6 +215,10 @@ final class KyuubiTHttpFrontendService(
         info(s"Starting and exposing JDBC connection at: jdbc:hive2://$connectionUrl/")
       }
       server.foreach(_.start())
+      // wait a moment for server thread starting ..
+      if (Utils.isTesting) {
+        Thread.sleep(3000)
```

Comment:
@yaooqinn  The jetty server state seems also not very accurate in a simple way.. I'm going back to add some sleep for test code path with HTTP mode. I'm not add this for binary mode for now and we can add if we see other flaky test in future.

Reply:
you can do it in test..

Reply:
changed..